dask-gateway/Dockerfile

```diff
@@ -21,7 +21,7 @@
 # psutils, a dependency of distributed, is currently the sole reason we have to
 # have this build stage.
 #
-FROM python:3.11-bullseye as build-stage
+FROM python:3.13-bullseye as build-stage
```

Comment:
Perfect!

Reply:
It could be that you need to refreeze the requirements.txt file based on requirements.in, now using a python 3.13 context - you updated the script doing that before - you could use it locally as well

Reply:
I refroze them now :)